src/style.css

```
@@ -0,0 +1,137 @@
+html,
+body,
+div,
+a,
+ul,
+li,
+header,
+nav,
+form,
+img,
+label,
+button {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+.form {
+  margin-top: 20px;
+  width: 100%;
+  background: hsl(0, 0%, 100%);
+  border: 1px solid hsl(210, 25%, 91%);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  position: relative;
+}
+
+.form--big {
+  height: 70px;
+}
+
+.form--small {
+  height: 42px;
+}
+
+.form__magnifying_glass {
+  position: absolute;
+  background-color: transparent;
+  padding: 0;
+  background-image: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  outline: none;
+  border: none;
+}
+
+.form__magnifying_glass--big {
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.form__magnifying_glass--small {
+  top: 15px;
+  left: 13px;
+  height: 11px;
+  width: 11px;
+}
+
+.form__input {
+  position: absolute;
+  height: 100%;
+  padding: 0;
+  font-family: Avenir, Helvetica, sans-serif;
```

the `font-family` can be set to `html` and here you can just use `inherit`
```suggestion
  font-family: inherit;
```